tests/model/SiteTreeTest.php

```diff
@@ -653,7 +653,62 @@ public function testIsSection() {
 		$this->assertTrue($staff->isSection());
 		$this->assertTrue($ceo->isSection());
 	}
-	
+
+	public function testURLSegmentAutoUpdate() {
+		Config::nest();
```

Comment:
Good spotting - was a remnant of the test code split out to `testURLSegmentAutoUpdateLocalized()`. Turns out you can't control `i18n::$current_locale` via the Config API...